scripts/merge_logs.py

```diff
@@ -0,0 +1,138 @@
+import shutil
+from typing import Union
+from pathlib import Path
+from datetime import datetime
+
+import pandas as pd
+
+
+def archive_logs(files: Union[str, list[str]], dest: str) -> None:
+    """Move single or list of files into
+
+    Parameters
+    ----------
+    files : Union[str, list[str]]
+        path of files to move
+    dest : str
+        Destinations to where the files will be move to
+    """
+
+    if isinstance(files, str):
+        files = files.split()
+
+    # checking if the directory exists
+    if Path(dest).is_file():
+        raise TypeError(e_msg)
+    elif not Path(dest).is_dir():
+        print("Archive folder does nto exists")
```

Comment:
```suggestion
        print("Archive folder does not exists")
```

Would it be helpful to create the directory then?